fdbclient/vexillographer/fdb.options

```diff
@@ -223,6 +223,8 @@ description is not currently required but encouraged.
             description="Enables tracing for this transaction and logs results to the client trace logs. The DEBUG_TRANSACTION_IDENTIFIER option must be set before using this option, and client trace logging must be enabled to get log output." />
     <Option name="transaction_logging_max_field_length" code="405" paramType="Int" paramDescription="Maximum length of escaped key and value fields."
             description="Sets the maximum escaped length of key and value fields to be logged to the trace file via the LOG_TRANSACTION option, after which the field will be truncated. A negative value disables truncation." />
+    <Option name="server_request_tracing" code="406"
+	    description="Sets a debug identifier for a transaction. When committed, this identifier triggers logging when each part of the transaction authority encounters it, which is helpful in diagnosing slowness in misbehaving clusters. The debug identifier is randomly generated. Call `get_transaction_debug_id` to read the identifier." />
```

Comment:
Does this function `get_transaction_debug_id` exist?

Reply:
It does not. I updated the text to reflect that and be more consistent with the current naming / functionality.